reporter/src/funTest/kotlin/TestData.kt

```diff
@@ -112,6 +112,14 @@ val ORT_RESULT = OrtResult(
                                     id = Identifier("NPM:@ort:declared-license:1.0")
                                 )
                             )
+                        ),
```

Comment:
Commit message nits:
- Please start capitalized after the colon in the title.
- Please end you sentence in the body with a dot.